src/test/java/AndreeaG/tests/tema6/NoSuchAlertException.java

```diff
@@ -0,0 +1,4 @@
+package AndreeaG.tests.tema6;
+
+public class NoSuchAlertException extends Throwable {
```

Comment:
You can delete this class as it's not used. Thanks!

Reply:
not even sure how I did that :) deleted it now